invenio_app_rdm/upgrade_scripts/fix_migrated_records_from_8_0_to_9_0.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+# Copyright (C) 2022 TU Wien.
+# Copyright (C) 2022 Graz University of Technology.
+#
+# Invenio-App-RDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Record migration fix script from InvenioRDM 8.0 to 9.0."""
+
+from click import secho
+from invenio_db import db
+from invenio_rdm_records.records.api import RDMDraft, RDMRecord
+
+
+def execute_upgrade_fix():
+    """Execute the upgrade fix from InvenioRDM 8.0 to 9.0."""
+
+    def update_parent(record):
+        """Update parent schema and parent communities for older records."""
+        new_parent_schema = "local://records/parent-v2.0.0.json"
+        record.parent["$schema"] = new_parent_schema
+        record.parent.setdefault("communities", {})
+
+    def update_record(record):
+        # skipping deleted records because can't be committed
+        if record.is_deleted:
+            return
+
+        try:
+            secho(f"Updating record : {record.pid.pid_value}", fg="yellow")
+
+            update_parent(record)
+
+            record.parent.commit()
+            record.commit()
+
+            secho(f"> Updated parent: {record.parent.pid.pid_value}", fg="green")
+            secho(f"> Updated record: {record.pid.pid_value}\n", fg="green")
```

Comment:
minor comment: this method's return value can be `None` (implicit) or a string (explicit) which is not very pythonic and is not return-consistent (Explicit is better than implicit.)

```suggestion
            secho(f"> Updated record: {record.pid.pid_value}\n", fg="green")
            Return None
```

Personally I am not a big fan of returning "negative" values in case of success but most of the migrations are written like this (return a `string` in case of error).